tests/main/ubuntu-core-refresh/task.yaml

```diff
@@ -52,11 +60,19 @@ execute: |
         snap revert "$TARGET_SNAP_NAME" &> revert.log
         MATCH "snapd is about to reboot the system" < revert.log
 
-        # use journalctl wrapper to grep only the logs collected while the test is running
-        while ! check_journalctl_log "Waiting for system reboot" -b; do
-            sleep 1
+        # Detect in the logs when the reboot can been triggered
```

Comment:
This reboot check loop and the check below could go into a helper function, but that's not critical.